diff -ur php-5.6.15-orig/configure php-5.6.15/configure
--- php-5.6.15-orig/configure	2016-01-25 23:05:45.887250715 +0500
+++ php-5.6.15/configure	2016-01-25 23:07:30.668648794 +0500
@@ -106781,7 +106781,7 @@
 EXTRA_LDFLAGS="$EXTRA_LDFLAGS $PHP_LDFLAGS"
 EXTRA_LDFLAGS_PROGRAM="$EXTRA_LDFLAGS_PROGRAM $PHP_LDFLAGS"
 
-PHP_BUILD_DATE=`date '+%Y-%m-%d'`
+PHP_BUILD_DATE="1970-01-01"
 
 cat >>confdefs.h <<_ACEOF
 #define PHP_BUILD_DATE "$PHP_BUILD_DATE"
diff -ur php-5.6.15-orig/configure.in php-5.6.15/configure.in
--- php-5.6.15-orig/configure.in	2016-01-25 23:05:44.603233580 +0500
+++ php-5.6.15/configure.in	2016-01-25 23:07:08.848357653 +0500
@@ -1275,7 +1275,7 @@
 EXTRA_LDFLAGS="$EXTRA_LDFLAGS $PHP_LDFLAGS"
 EXTRA_LDFLAGS_PROGRAM="$EXTRA_LDFLAGS_PROGRAM $PHP_LDFLAGS"
 
-PHP_BUILD_DATE=`date '+%Y-%m-%d'`
+PHP_BUILD_DATE="1970-01-01"
 AC_DEFINE_UNQUOTED(PHP_BUILD_DATE,"$PHP_BUILD_DATE",[PHP build date])
 
 case $host_alias in
diff -ur php-5.6.15-orig/ext/standard/info.c php-5.6.15/ext/standard/info.c
--- php-5.6.15-orig/ext/standard/info.c	2016-01-25 23:05:45.179241268 +0500
+++ php-5.6.15/ext/standard/info.c	2016-01-25 23:12:28.252623777 +0500
@@ -942,7 +942,7 @@
 		php_info_print_box_end();
 		php_info_print_table_start();
 		php_info_print_table_row(2, "System", php_uname );
-		php_info_print_table_row(2, "Build Date", __DATE__ " " __TIME__ );
+		php_info_print_table_row(2, "Build Date", "Jan  1 1970 00:00:01");
 #ifdef COMPILER
 		php_info_print_table_row(2, "Compiler", COMPILER);
 #endif
diff -ur php-5.6.15-orig/sapi/cgi/cgi_main.c php-5.6.15/sapi/cgi/cgi_main.c
--- php-5.6.15-orig/sapi/cgi/cgi_main.c	2016-01-25 23:05:46.111253703 +0500
+++ php-5.6.15/sapi/cgi/cgi_main.c	2016-01-25 23:12:04.580307054 +0500
@@ -2231,9 +2231,9 @@
 								SG(request_info).no_headers = 1;
 							}
 #if ZEND_DEBUG
-							php_printf("PHP %s (%s) (built: %s %s) (DEBUG)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, __DATE__, __TIME__, get_zend_version());
+							php_printf("PHP %s (%s) (built: Jan  1 1970 00:00:01) (DEBUG)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, get_zend_version());
 #else
-							php_printf("PHP %s (%s) (built: %s %s)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, __DATE__, __TIME__, get_zend_version());
+							php_printf("PHP %s (%s) (built: Jan  1 1970 00:00:01)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, get_zend_version());
 #endif
 							php_request_shutdown((void *) 0);
 							fcgi_shutdown();
diff -ur php-5.6.15-orig/sapi/cli/php_cli.c php-5.6.15/sapi/cli/php_cli.c
--- php-5.6.15-orig/sapi/cli/php_cli.c	2016-01-25 23:05:46.091253437 +0500
+++ php-5.6.15/sapi/cli/php_cli.c	2016-01-25 23:11:31.327862162 +0500
@@ -693,8 +693,8 @@
 				goto out;
 
 			case 'v': /* show php version & quit */
-				php_printf("PHP %s (%s) (built: %s %s) %s\nCopyright (c) 1997-2015 The PHP Group\n%s",
-					PHP_VERSION, cli_sapi_module.name, __DATE__, __TIME__,
+				php_printf("PHP %s (%s) (built: Jan  1 1970 00:00:01) %s\nCopyright (c) 1997-2015 The PHP Group\n%s",
+					PHP_VERSION, cli_sapi_module.name,
 #if ZEND_DEBUG && defined(HAVE_GCOV)
 					"(DEBUG GCOV)",
 #elif ZEND_DEBUG
diff -ur php-5.6.15-orig/sapi/fpm/fpm/fpm_main.c php-5.6.15/sapi/fpm/fpm/fpm_main.c
--- php-5.6.15-orig/sapi/fpm/fpm/fpm_main.c	2016-01-25 23:05:46.099253543 +0500
+++ php-5.6.15/sapi/fpm/fpm/fpm_main.c	2016-01-25 23:11:06.139525160 +0500
@@ -1752,9 +1752,9 @@
 				SG(request_info).no_headers = 1;
 
 #if ZEND_DEBUG
-				php_printf("PHP %s (%s) (built: %s %s) (DEBUG)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, __DATE__,        __TIME__, get_zend_version());
+				php_printf("PHP %s (%s) (built: Jan  1 1970 00:00:01) (DEBUG)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, get_zend_version());
 #else
-				php_printf("PHP %s (%s) (built: %s %s)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, __DATE__, __TIME__,      get_zend_version());
+				php_printf("PHP %s (%s) (built: Jan  1 1970 00:00:01)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name,      get_zend_version());
 #endif
 				php_request_shutdown((void *) 0);
 				fcgi_shutdown();
diff -ur php-5.6.15-orig/sapi/litespeed/lsapi_main.c php-5.6.15/sapi/litespeed/lsapi_main.c
--- php-5.6.15-orig/sapi/litespeed/lsapi_main.c	2016-01-25 23:05:46.083253330 +0500
+++ php-5.6.15/sapi/litespeed/lsapi_main.c	2016-01-25 23:10:40.487181936 +0500
@@ -840,9 +840,9 @@
             case 'v':
                 if (php_request_startup(TSRMLS_C) != FAILURE) {
 #if ZEND_DEBUG
-                    php_printf("PHP %s (%s) (built: %s %s) (DEBUG)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, __DATE__, __TIME__, get_zend_version());
+                    php_printf("PHP %s (%s) (built: Jan  1 1970 00:00:01) (DEBUG)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, get_zend_version());
 #else
-                    php_printf("PHP %s (%s) (built: %s %s)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, __DATE__, __TIME__, get_zend_version());
+                    php_printf("PHP %s (%s) (built: Jan  1 1970 00:00:01)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, get_zend_version());
 #endif
 #ifdef PHP_OUTPUT_NEWAPI
                     php_output_end_all(TSRMLS_C);
diff -ur php-5.6.15-orig/sapi/milter/php_milter.c php-5.6.15/sapi/milter/php_milter.c
--- php-5.6.15-orig/sapi/milter/php_milter.c	2016-01-25 23:05:46.083253330 +0500
+++ php-5.6.15/sapi/milter/php_milter.c	2016-01-25 23:10:03.094682567 +0500
@@ -1113,7 +1113,7 @@
 				}
 				SG(headers_sent) = 1;
 				SG(request_info).no_headers = 1;
-				php_printf("PHP %s (%s) (built: %s %s)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, __DATE__, __TIME__, get_zend_version());
+				php_printf("PHP %s (%s) (built: Jan  1 1970 00:00:01)\nCopyright (c) 1997-2015 The PHP Group\n%s", PHP_VERSION, sapi_module.name, get_zend_version());
 				php_output_teardown();
 				exit(1);
 				break;
diff -ur php-5.6.15-orig/sapi/phpdbg/phpdbg.c php-5.6.15/sapi/phpdbg/phpdbg.c
--- php-5.6.15-orig/sapi/phpdbg/phpdbg.c	2016-01-25 23:05:46.107253651 +0500
+++ php-5.6.15/sapi/phpdbg/phpdbg.c	2016-01-25 23:09:27.962213803 +0500
@@ -1199,10 +1199,8 @@
 				sapi_startup(phpdbg);
 				phpdbg->startup(phpdbg);
 				printf(
-					"phpdbg %s (built: %s %s)\nPHP %s, Copyright (c) 1997-2015 The PHP Group\n%s",
+					"phpdbg %s (built: Jan  1 1970 00:00:01)\nPHP %s, Copyright (c) 1997-2015 The PHP Group\n%s",
 					PHPDBG_VERSION,
-					__DATE__,
-					__TIME__,
 					PHP_VERSION,
 					get_zend_version()
 				);
